.github/pull_request_template.md

```diff
@@ -9,11 +9,21 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+These should be followed unless you're dealing with an emergency. In an emergency, use your best judgment. 
+
+- [ ] I've tested the code as an end user would.
+- [ ] For all new functionality, I have added thorough, applicable tests that validate my changes
+- [ ] For improvements to existing functionality, I've followed the [campsite rule](https://wolfgang-ziegler.com/blog/Development-Practice-The-Campsite-Rule.aspx) (made it better than it was). This is for tests or other code behavior. Don't overinvest, but also don't underinvest in "campsite" practices to improve the area you are working in. Also it can be good to separate the commits for your cleanup work and your functional changes. 
```

Comment:
This is a great callout, but a little wordy for . Maybe we add something about this in the [development standards doc](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git), and link out to that instead for more color?

Reply:
Great idea. I've prepared a commit there, but don't seem to have write permissions. Will submit a separate PR for that and tidy this up. Good suggestion!

Reply:
Added separate PR to incorporate this suggestion: https://github.com/encodium/.github-private/pull/10